src/document.h

```diff
@@ -1126,6 +1126,26 @@ struct Document {
                 ZoomOutIfNoGrid(dc);
                 return nullptr;
 
+            case A_BACKSPACE_WORD:
+                if (c && selected.TextEdit()) {

```

Comment:
these two actions can be moved to below around line 1569 where it says `if (!selected.TextEdit()) return _(L"only works in cell text mode");`, so this `if` can be omitted.